content/en/index.md

```diff
@@ -10,13 +10,10 @@ description: 'Preact is a fast 3kB alternative to React with the same modern API
     <h1>
         <logo height="1.5em" title="Preact" text inverted>Preact</logo>
     </h1>
-    <p>Fast 3kB alternative to React with the same modern API.</p>
+    <p class="tagline">Fast 3kB alternative to React with the same modern API.</p>
```

Comment:
Its better if we have the `tagline` in the next line cuz right now the `logo` & `Preact and tagline` looks a bit misaligned to me.

(maybe that's just me)

Reply:
Looks like that's what the majority on twitter prefers too :+1: 

https://mobile.twitter.com/marvinhagemeist/status/1233595672751935491